index.js

```diff
@@ -153,7 +154,8 @@ async function handleIncreaseScores (req, res, pgPool, signerAddresses, logger)
 }
 
 async function handlePaidScheduledRewards (req, res, pgPool, signerAddresses, logger) {
-  const body = JSON.parse(await getRawBody(req, { limit: '1mb' }))
+  const rawBody = await getRawBody(req, { limit: '1mb' })
+  const body = JSON.parse(rawBody.toString())
```

Comment:
Looking at the type declaration of `rawBody`, would the following work?

```js
const body = JSON.parse(await getRawBody(req, { limit: '1mb', encoding: 'utf-8' }))
```

See https://github.com/stream-utils/raw-body/blob/f62e660e7c50891844f5615de075ab145c1f6129/index.d.ts#L75-L83

Reply:
Great job on digging into the type definitions 🫶